chimney/src/main/scala/io/scalaland/chimney/internal/macros/DslWhiteboxMacros.scala

```diff
@@ -1,95 +1,109 @@
 package io.scalaland.chimney.internal.macros
 
+import io.scalaland.chimney.internal.utils.MacroUtils
+
 import scala.reflect.macros.whitebox
 
-trait DslWhiteboxMacros {
+trait DslWhiteboxMacros extends MacroUtils {
 
   val c: whitebox.Context
 
   import c.universe._
 
-  def expandWithFieldConst[From: c.WeakTypeTag, To: c.WeakTypeTag, C: c.WeakTypeTag](
+  def expandFieldConst[
+      From: c.WeakTypeTag,
+      To: c.WeakTypeTag,
+      C: c.WeakTypeTag
+  ](
       fieldName: Name,
       value: c.Tree
   ): c.Tree = {
-    val From = weakTypeOf[From]
-    val To = weakTypeOf[To]
-    val C = weakTypeOf[C]
-
-    val fieldNameConst = Constant(fieldName.decodedName.toString)
-    val fieldNameLit = Literal(fieldNameConst)
-    val singletonFieldTpe = c.internal.constantType(fieldNameConst)
-    val newCfgTpe = tq"_root_.io.scalaland.chimney.internal.TransformerCfg.FieldConst[$singletonFieldTpe, $C]"
-    val fn = TermName(c.freshName("ti"))
+    q"""
+      ${c.prefix.tree}
+        .__addOverride(${fieldName.toNameLiteral}, $value)
```

Comment:
How about  some extension methods allowing something like:
```scala
c.prefix.tree
  .addOverride(fieldName.toNameLiteral, value)
  .refineConfigFieldConst(fieldName.toSingletonTpe, weakTypeOf[C]) // maybe sth better?
```
it would be easier to refactor in the future if any update/rename/change could be less stringly.

I mean here and in the following quasiquites.

Reply:
If you mean extension methods on trees (not adding this stuff to public api), I'm fine with such refactor 👍

Reply:
done ✅